backend/seeds/seed.py

```diff
@@ -0,0 +1,14 @@
+import json
+from database.database import get_db
+from symptoms.models import Symptom
+
+# Using `next` here because get_db() returns a generator
+session = next(get_db())
```

Comment:
@pyritewolf Do you think this `next` stuff is ok?

Reply:
Hm. Hmm. Idk. I'm not 100% sure on this - I'm as new as you are to SQLAlchemy - but shouldn't it be just `get_db` without the `()`? I'm just looking at what other we do in some other files, but frankly your guess is as good as mine.